.github/render-action/generate.sh

```diff
@@ -31,6 +33,7 @@ function convert_markdown_to_asciidoc {
 
     echo "Converting ${md_filename} to ${adoc_filepath}/${adoc_filename}..."
     kramdoc \
+      -a author="Wyrmworks Publishing"  \
```

Comment:
This is used on the title page. We could also change it to something else of course; depending on what we want to show.

This is an example of what that may look like:
<img width="724" height="943" alt="image" src="https://github.com/user-attachments/assets/8ec727a6-1b98-4c8e-95b8-39fa5061add1" />